pyramid_swagger/tween.py

```diff
@@ -273,6 +301,8 @@ def load_settings(registry):
     return Settings(
         schema=schema,
         swagger_handler=build_swagger_handler(registry.settings, schema),
+        swagger12_handler=build_swagger12_handler_wrapper(
```

Comment:
This caching is done so that for every request no new swagger12_handler is created. This will be `None` if v1.2 is not in the enabled swagger versions. This will be duplicated in case only enabled swagger version was v1.2 but i think it is OK as it simplifies the code.